assets/js/search.js

```diff
@@ -30,17 +29,28 @@ setDateToday();
     )
 })(jQuery);
 
-$(document).ready(function CheckPreviousLocation() {
-  const elem = document.getElementById("location");
-  if (sessionStorage.getItem("locationPerm")) {
-    console.log(
-      `Set the Location based on PageLoad...` +
-        sessionStorage.getItem("locationPerm")
-    );
-    elem.value = sessionStorage.getItem("locationPerm");
-  }
+document.getElementById("logout-confirm-btn").addEventListener(
+    "click", logOut
+)
+
```

Comment:
Do this part above was removed with my PR ?

Reply:
Yes